docs/docs-content/clusters/edge/trusted-boot/deployment-day2/upgrade-cluster.md

```diff
@@ -0,0 +1,98 @@
+---
+sidebar_label: "Update Cluster"
+title: "Update Cluster with Trusted Boot"
+description: "Learn about Trusted Boot."
+hide_table_of_contents: false
+sidebar_position: 60
+tags: ["edge"]
+---
+
+If you need to make changes to a deployed Edge cluster with Trusted Boot enabled without touching the OS or the
```

Comment:
```suggestion
If you need to make changes to a deployed Edge cluster with Trusted Boot enabled without modifying the Operating System (OS) or the
```

Replaced "touching" to the more formal form better fitted in technical documentation.